### PR TITLE
OpenSLR Sundanese ASR

### PR DESCRIPTION
**Pull request by peter-yh-wu**
ASR for [OpenSLR Large Sundanese ASR training data set](http://www.openslr.org/36/)

### CODE REVIEW

File: egs2/sunda/asr1/setup.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+../../TEMPLATE/asr1/setup.sh
```
Comment (ftshijt) [inline egs2/sunda/asr1/setup.sh:1]: We do not need this file for specific recipe

File: egs2/sunda/asr1/run.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+asr_config=conf/train_asr.yaml
+inference_config=conf/decode_asr.yaml
+
+ngpu=1
+
+# NOTE started in stage 1 for alam5
+
+./asr.sh \
+    --stage 2 \
```
Comment (ftshijt) [inline egs2/sunda/asr1/run.sh:22]: Please set it as the initial setting (e.g., stage 1 )

File: egs2/sunda/asr1/copy_files.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cp -r ../../alam6_wld/asr1/data/$1_test data/
```
Comment (ftshijt) [inline egs2/sunda/asr1/copy_files.sh:3]: Do we need this?

File: egs2/sunda/asr1/local/score.sh [hunk 4]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```
Comment (ftshijt) [inline egs2/sunda/asr1/local/score.sh:1]: If this is the same as https://github.com/espnet/espnet/blob/master/egs2/open_li52/asr1/local/score.sh you can use soft link for this

File: egs2/sunda/asr1/run_eval.sh [hunk 5]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+lid=false # whether to use language id as additional label
```
Comment (ftshijt) [inline egs2/sunda/asr1/run_eval.sh:8]: Do we need this?

File: egs2/sunda/asr1/run_wld.sh [hunk 6]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+lid=false # whether to use language id as additional label
+
+train_set="train"
```
Comment (ftshijt) [inline egs2/sunda/asr1/run_wld.sh:10]: I don't think this shell script is necessary. If you think it is important, please share the reason

**Review by ftshijt — CHANGES_REQUESTED** [review]
Thanks a lot. Just some minor issues

**Comment by ftshijt** [comment]
Hi, Just a follow up to Shinji's advice, could you upload your model to espnet_model_zoo? You can check the steps here https://github.com/espnet/espnet_model_zoo

**Comment by ftshijt** [comment]
Meanwhile, you may add a README in the Sundanese for results and pre-trained model (after you get the link from model_zoo

File: egs2/su_openslr36/asr1/local/sunda_data_prep.py [hunk 7]
```diff
@@ -0,0 +1,90 @@
+import argparse
```
Comment (sw005320) [inline egs2/su_openslr36/asr1/local/sunda_data_prep.py:1]: Please add an appropriate header (shebang and copyright)

**Review by sw005320 — APPROVED** [review]

**Comment by ftshijt** [comment]
Since you already upload the model in model zoo, could you please also update the pre-trained link and results in RESULTS.md  (similar to https://github.com/espnet/espnet/tree/master/egs2/open_li52/asr1/README.md )

**Review by ftshijt — APPROVED** [review]

**Pull request merged**